backend/subroker/views.py

```diff
@@ -312,28 +312,118 @@ def group_add_user(request, group_id):
 def content_list(request, content_id):
     return HttpResponse("content search")
 
+#search from the-movie-db by search string
+def content_search(request, search_str):
+    if request.method == 'GET':
+        # keep track of how many times we've retried
+        MAX_RETRIES = 2
+        attempt_num = 0  
+        while attempt_num < MAX_RETRIES:
+            url = 'https://api.themoviedb.org/3/search/movie'
+            params={'api_key':'e50a928b29c2b7ed1f82ba2e6ce4982c','query':search_str.replace(" ","+")}
```

Comment:
@muhwagwa It would be better to extract api_key as a global variable.

Reply:
Sure! But could you help me out with setting global variables next time? still having difficulty with it.